rasa_nlu/extractors/crf_entity_extractor.py

```diff
@@ -81,14 +90,31 @@ def __init__(self, component_config=None, ent_tagger=None):
 
         self._validate_configuration()
 
+        self._check_pos_features_and_spacy()
+
+    def _check_pos_features_and_spacy(self):
+        import itertools
+        features = self.component_config.get("features", [])
+        fts = set(itertools.chain.from_iterable(features))
+        self.pos_features = ('pos' in fts or 'pos2' in fts)
+        if self.pos_features:
+            self._check_spacy()
+
+    @staticmethod
+    def _check_spacy():
+        if spacy is None:
+            raise ImportError(
+                'Failed to import `spaCy`. '
+                '`spaCy` is required for POS features.')
```

Comment:
please provide some more info, how should the user fix this issue? e.g. documentation link